assets/js/protocol/protocol-navigation.js

```diff
@@ -169,14 +171,36 @@ if (typeof window.Mzp === 'undefined') { // eslint-disable-line block-scoped-var
             }
         }
     };
+    /**
+     * use Intersection Observer API to keep track of when the mobile
+     * nav menu is displayed to handle aria roles better
+     */
+
+    Navigation.menuButtonVisible = function (callback) {
+        var observer = new IntersectionObserver(function(entries) {
```

Comment:
Might this throw an error in older browsers?

Reply:
ack - you're right. Not supported in IE and browsers pre-2017ish - I'll need to look how to cover this in older browsers. Thanks for calling it out

Reply:
It might be possible for you to use this and add a feature detect maybe, unless there are other options you think of